src/vivarium_census_prl_synth_pop/tools/cli.py

```diff
@@ -66,14 +66,9 @@ def make_artifacts(
 
 @click.command()
 @click.argument("output_dir", type=click.Path(exists=True))
-@click.option(
-    "-V",
-    "--label-version",
-    type=str,
-    default=None,
-    show_default=True,
-    help="Provide a version number for final results. "
-    "Version should follow format of '#.#.#'.",
+@click.argument(
```

Comment:
I originally had the same thought. My opinion is we have to (at any point) determine a version number to put in the metadata file. My reason for making it a required arg is since we need to it to the metadata file, we should be passing something thats in a consistent format. In pseudopeople we would then have to compare data versions for 2.0.0 or for SOME_RANDOM_DATE..